### PR TITLE
Major structural update: Docusaurus v3 + Social VM integration

### PR DESCRIPTION
**Pull request by gagdiez**
- Updated `docusaurus` to `v3`
- Removed lots of unnecesary / legacy files (e.g. `copy-button.css`, `near.min.css`)
- Re-structured folders to be closer to a vanilla docusaurus project
- Added the Social VM to `docusaurus`, so now we can import `Widgets` directly
- Reworked components system

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -213,13 +208,17 @@ ${content}`, // <-- this last part adds in the rest of the content, which would
...
+          position: "right"
+        },
       ],
     },
     image: 'img/near_logo.svg',
     footer: {
-      links: [], 
-      copyright: 'Copyright © 2021 NEAR Protocol',
+      links: [],
+      copyright: "Copyright © 2023 NEAR Protocol",
```
Comment (thisisjoshford) [inline website/docusaurus.config.js:221]: We'll need to implement lint rules docs around quotes as I've noticed our respective settings with quotations have been conflicting.

File: website/package.json [hunk 2]
```diff
@@ -1,55 +1,70 @@
...
     "full-test": "./test-links.sh test",
-    "examples": "docusaurus-examples",
-    "start": "node ./utils/getChangelogs.js && docusaurus start",
-    "build": "node ./utils/getChangelogs.js && docusaurus build",
-    "publish-gh-pages": "docusaurus-publish",
-    "write-translations": "docusaurus write-translations",
-    "write-heading-ids": "docusaurus write-heading-ids",
-    "version": "docusaurus-version",
-    "rename-version": "docusaurus-rename-version",
+    "start": "docusaurus start",
```
Comment (thisisjoshford) [inline website/package.json:7]: If we remove `getChangelogs.js` script here we need to implement another way of checking for a new release report.

File: website/src/components/social-widget.js
Comment (thisisjoshford) [inline website/src/components/social-widget.js:1]: 🔥 Stoked to get the VM integrated w/ docs 💪

File: website/src/theme/Gateway/wallet-selector.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  const setAuth = useWallet(store => store.setAuth);
+  const setLogActions = useWallet(store => store.setLogActions);
+  const setMethods = useWallet(store => store.setMethods);
+  const setStoreSelector = useWallet(store => store.setStoreSelector);
+  const [selector, setSelector] = useState(undefined);
+
+  useEffect(() => {
+    const selector = setupWalletSelector({
+      network: networkId,
+      modules: [setupMyNearWallet(), setupHereWallet()]
```
Comment (thisisjoshford) [inline website/src/theme/Gateway/wallet-selector.js:34]: any reason why we are limiting to these two wallets?

**Review by thisisjoshford — APPROVED** [review]
Awesome job @gagdiez ! 

Love the file structure cleanup, upgrade to v3, and most importantly VM integration! This feature will be killer as we move towards interactive docs. 💪

File: website/src/theme/Gateway/wallet-selector.js (hunk 3 above)
Comment (gagdiez) [reply website/src/theme/Gateway/wallet-selector.js:34]: these are the ones that I tested and know that works

File: website/package.json (hunk 2 above)
Comment (gagdiez) [reply website/package.json:7]: added a script to update it, but didn't add it on every start. We should automatize that with actions maybe.

**Pull request merged**